src/local-mode.ts

```diff
@@ -142,6 +148,39 @@ export class LocalModeManager {
 		);
 	}
 
+	private async collectContainerResources(
+		nameOrId: string,
+	): Promise<EngineSnapshot> {
+		const inspectInfo = await this.docker.getContainer(nameOrId).inspect();
+		return new EngineSnapshot(
+			[inspectInfo.Id],
+			[inspectInfo.Image],
+			inspectInfo.Mounts.filter(m => m.Name !== undefined).map(m => m.Name!),
```

Comment:
I think this should be `== null`. The docker daemon can sometimes return `null` for various fields, or omit the field altogether. The best thing to do with using the docker api is to always use `== null` to cover both instances.

Reply:
Good to know, thanks.

Reply:
Changed.